plugins/introduction.rst

```diff
@@ -124,22 +125,22 @@ plugins or configure a different client. For example::
         /**
          * Build the HTTP client to talk with the API.
          *
-         * @param string     $user   Username
-         * @param string     $pass   Password
-         * @param HttpClient $client Base HTTP client
+         * @param string     $user    Username
+         * @param string     $pass    Password
+         * @param Plugin[]   $plugins List of additional plugins to use
+         * @param HttpClient $client  Base HTTP client
          *
          * @return HttpClient
          */
-        public static function create($user, $pass, HttpClient $client = null)
+        public static function create($user, $pass, $plugins = [], HttpClient $client = null)
```

Comment:
fixed. seems the new github interface does not automatically hide comments that are obsolete anymore